entities/models/tenant.go

```diff
@@ -75,6 +75,15 @@ const (
 
 	// TenantActivityStatusFROZEN captures enum value "FROZEN"
 	TenantActivityStatusFROZEN string = "FROZEN"
+
+	// TenantActivityStatusFREEZING captures enum value "FREEZING"
+	TenantActivityStatusFREEZING string = "FREEZING"
+
+	// TenantActivityStatusUNFREEZING captures enum value "UNFREEZING"
+	TenantActivityStatusUNFREEZING string = "UNFREEZING"
+
+	// TenantActivityStatusUNFROZEN captures enum value "UNFROZEN"
+	TenantActivityStatusUNFROZEN string = "UNFROZEN"
```

Comment:
**suggestion (non-blocking):** i'd say we can remove UNFROZEN to not confuse the users given that it's internal usage and shouldn't be returned to the user anyway

Reply:
As it stands, will it be returned if `GET /v1/schema/{className}/tenants` is queried? Looking at the source, it appears that `ActivityStatus` is assigned to at `usecases/schema/tenant.go#L298` as so:
```go
ActivityStatus: schema.ActivityStatus(ss.Physical[name].Status),
```
without checks for the status itself and whether it is `UNFROZEN`. Is there somewhere else that this value would be converted into something visible to the user?

Reply:
Resolved by change to `stable/v1.26` removing `UNFROZEN` from the internals